Add permissions explanation to import (#5799)
treeverse/lakeFS: docs/howto/import.md

### Prerequisites

#### User Permissions

> The lakeFS user running the import must have the following permissions:
> `fs:WriteObject`, `fs:CreateMetaRange`, `fs:CreateCommit` and 
> `fs:ImportFromStorage`. The first 3 permissions are available by default to 

Reviewer: ```suggestion
To run import you need the following permissions:
`fs:WriteObject`, `fs:CreateMetaRange`, `fs:CreateCommit`, and 
`fs:ImportFromStorage`. The first 3 permissions are available by default to 
```

(Note the extra "Oxford comma".  I think @YaelRiv had a rule to use those, but of course we can change it.  @rmoff , any preference?

users in the default Developers group ([RBAC](../reference/rbac.md)) or the 
Writers group ([ACL](../reference/access-control-lists.md)). The last 
permission enables the user to import data from any location of the storage 

> provider that lakeFS has access to. Thus, it's only available to users in the 

Reviewer 2: Maybe we should emphasize this sentence. Or even add it as some kind of warning on top

Author: IMO it's bad enough that we have to start an how-to page with permissions adjustments for some non-trivial use-cases. Adding warnings on top of that gets them further from the value..

> default SuperUsers group ([RBAC](../reference/rbac.md)) or the Super 

Reviewer: I think that we should rephrase to make it more procedural.  Perhaps something like:

> ... only available to users in group Supers (on ACL) or SuperUsers (on RBAC)

?  And similarly for all other alternatives.

Author: done.

> provider that lakeFS has access to. Thus, it's only available to users in the 
> default SuperUsers group ([RBAC](../reference/rbac.md)) or the Super 
> group ([ACL](../reference/access-control-lists.md)). RBAC installations can 

Author: ```suggestion
provider that lakeFS has access to. Thus, it's only available to users in group Supers ([ACL](../reference/access-control-lists.md)) or SuperUsers ([RBAC](../reference/rbac.md)). RBAC installations can 
```

> add that permission to any group, including Developers.

Reviewer: ```suggestion
modify policies to add that permission to any group, such as Developers.
```
